client/src/components/ProfileListings/ProfileCard/ProfileCard.tsx

```diff
@@ -17,45 +20,44 @@ export default function ProfileCard({ profile }: Props): JSX.Element {
   const classes = useStyles();
 
   return (
-    <Paper elevation={8}>
-      <Box padding={6} justifySelf="center">
-        <Grid container direction="column" alignItems="center" spacing={2}>
-          <Avatar className={classes.avatar} src={`/mock/${profile.photoUrl}`} />
-          <Typography variant="h4">{`${profile.firstName} ${profile.lastName}`}</Typography>
-          <Typography variant="h5">Loving pet sitter</Typography>
-          <Grid item>
-            <Box p={1}>
-              <StarIcon className={classes.star} />
-              <StarIcon className={classes.star} />
-              <StarIcon className={classes.star} />
-              <StarIcon className={classes.star} />
-              <StarIcon className={classes.star} />
-            </Box>
-          </Grid>
-
-          <Box maxWidth="300px">
-            <Typography variant="body1" paragraph={true} align="center">
-              Dog sitting, cat sitting, pocket pet and bird care
-            </Typography>
-          </Box>
-        </Grid>
-      </Box>
-      <Divider />
-      <Box padding={3}>
-        <Grid container justify="space-between" alignItems="center">
-          <Grid item>
-            <Grid container alignItems="center">
-              <LocationOnIcon className={classes.location} fontSize="large" />
-              <Typography variant="body1">{profile.address}</Typography>
+    <Card elevation={8}>
+      <Link href={`/profiles/${profile.userId}`} color="inherit" underline="none">
+        <CardActionArea>
```

Comment:
```suggestion
        <CardActionArea component={Link} to="...">
```

Please ensure you are using react-router's Link and not the one from MUI